### ISSUE TITLE
The metric `process_resident_memory_bytes` was not calculated correctly

### ISSUE THREAD

**Issue by zhengtianbao** [VictoriaMetrics/VictoriaMetrics#6457]
### Describe the bug

When I use vm cluster deployed in KyLin OS and use grafana to monitor VM metrics, I find the `process_resident_memory_bytes` is less than `process_resident_memory_anon_bytes`, which means there must be something wrong.

Then I checked the code:

https://github.com/VictoriaMetrics/metrics/blob/5e5f209f79e7243b45ca978e65ccfc103daee01e/process_metrics_linux.go#L83

The hard-coded number `4096` means memory page size, but at KyLin OS the default value is 65536.

```
$ getconf PAGESIZE
65536
``` 

### To Reproduce

Start vm at an OS that sets page_size not equal 4096.

### Version

```
$ ./vmstorage-prod --version
vmstorage-20230407-011351-tags-v1.90.0-cluster-0-g64cf8fe49
```

**Comment by hagen1778** [comment]
@zhengtianbao I'm re-opening this issue because it still requires additional work to do. It will be closed only after the changes will be included to VM release.

@f41gh7, shall we update the dependency of the metrics package?

**Comment by valyala** [comment]
VictoriaMetrics components must properly calculate the value for `process_resident_memory_bytes` metric depending on the OS page size starting from [v1.102.0-rc2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.102.0-rc2). Closing the issue as fixed.

**Comment by valyala** [comment]
FYI, this issue has been fixed in [v1.97.6 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.97.6). The bugfix will be included in the upcoming release too.
